lucetc/src/function.rs

```diff
@@ -22,26 +22,33 @@ pub struct FuncInfo<'a> {
     module_decls: &'a ModuleDecls<'a>,
     codegen_context: &'a CodegenContext,
     count_instructions: bool,
-    scope_costs: Vec<u32>,
+    scope_costs: Vec<(u32, bool)>,
```

Comment:
I'd prefer to see this as a struct with named fields, rather than a tuple. Seeing this `bool` here alone is a head-scratcher

Reply:
Yup, that makes it much clearer, thanks!